packages/@adobe/spectrum-css-temp/components/calendar/skin.css

```diff
@@ -114,4 +114,12 @@ governing permissions and limitations under the License.
       color: var(--spectrum-calendar-day-text-color-disabled);
     }
   }
+
+  &.is-unavailable {
+    &,
```

Comment:
lol, specificity strikes again?

Reply:
Copied from `is-disabled` above. This needs to override that.